cmd/sync/sync.go

```diff
@@ -48,6 +48,8 @@ var githubUser = flag.String("github-user", "", "Use this github user to submit
 var githubPAT = flag.String("github-pat", "", "Submit the PR with this GitHub PAT, if specified.")
 var githubPATReviewer = flag.String("github-pat-reviewer", "", "Approve the PR and turn on auto-merge with this PAT, if specified. Required, if github-pat specified.")
 
+var azdoDncengPAT = flag.String("azdo-dnceng-pat", "", "Use this Azure DevOps PAT to authenticate to dnceng project HTTPS Git URLs.")
```

Comment:
IMO passing secrets via command line is not secure, as they can be leaked in many ways. Could we use an environmental variable instead, such as `AZDO_DNCENG_PAT?